content/en/building/targets/targets.md

```diff
@@ -1,16 +1,16 @@
 ---
-title: "Building Target Widgets"
+title: "Target Widgets"
 linkTitle: Targets
```

Comment:
```suggestion
linkTitle: Targets Widgets
```

I suggest we specify the `Widgets` word, as otherwise is becomes Targets / Targets hierarchy and it is a bit confusing.